docs/hw/spec-overview.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Build Armbian image
+parent: Hardware
+nav_order: 210
+---
+
+## Specifications Overview
+
+Platform: RockPro64 4GB DDR4 RAM
```

Comment:
Lines should be prefixed with `*` to form a list, otherwise everything get concatenated into one long blob